photon-api/src/main/scala/com/linkedin/photon/ml/optimization/SingleNodeOptimizationProblem.scala

```diff
@@ -91,9 +91,10 @@ protected[ml] class SingleNodeOptimizationProblem[Objective <: SingleNodeObjecti
 
     val normalizationContext = optimizer.getNormalizationContext
     val (optimizedCoefficients, stateTracker, _) = optimizer.optimize(objectiveFunction, initialModel.coefficients.means)(input)
-    val optimizedVariances = computeVariances(input, optimizedCoefficients)
+    val origOptimizedCoef = normalizationContext.value.coefToOriginalSpace(optimizedCoefficients)
+    val origOptimizedVar = computeVariances(input, origOptimizedCoef)
```

Comment:
Thanks for fixing this bug. We do need to transform the coefficients back to the original space then compute the variance.

Reply:
Yes, this is the correct and easiest way to compute variance when feature normalization is involved.